lib/modules/dosomething/dosomething_campaign/includes/CampaignController.php

```diff
@@ -2,6 +2,15 @@
 
 class CampaignController {
 
-  // @TODO: handle main app controller-y things such as CRUD in here.
+  /**
+   * Update the specified Campaign in storage.
+   *
+   * @param  int  $id
+   * @return void
+   */
+  function update($id) {
+    cache_clear_all('campaigns_api_request', 'cache_dosomething_api', TRUE);
```

Comment:
this clears the entire api cache when any node is updated. I don't think we want that....